### PR TITLE
#519 Project estimate

### PR DESCRIPTION
**Pull request by carlosmiranda**

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, someone will review your pull request soon

**Comment by davvd** [comment]
@longtimeago it's yours, please review

File: est/001.est [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  1: Website UI design
+  2: API design 
+  3: User addition, modification, and deletion of repos
+  4: Automatic deployment of user repos
+  5: Billing of resource usage
+  6: Resource scaling functionality
+  7: Integration testing
+champions:
+  2:
+    worst-case: 40
```
Comment (longtimeago) [inline est/001.est]: I'm really curious, where did you get these numbers?

Comment (carlosmiranda) [inline est/001.est]: I applied the techniques described in https://github.com/teamed/est. I think every estimator can have very different numbers, which is integral to the methodology itself. :)

Comment (longtimeago) [inline est/001.est]: Yep, I know estimations are very personal :)

**Comment by longtimeago** [comment]
@rultor please merge

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by carlosmiranda** [comment]
@rultor merge

**Comment by longtimeago** [comment]
@rultor deploy?

**Comment by carlosmiranda** [comment]
@rultor deploy please

**Comment by yegor256** [comment]
@rultor deploy pls

**Comment by davvd** [comment]
@longtimeago thanks, paid, **16 mins** to your account, payment ID is `53325099`... extra minutes for review comments (c=1)... +16 added to your rating, at the moment it is: [+2866](http://www.netbout.com/b/35024?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**